scripts/testdata/git.ts

```diff
@@ -0,0 +1,194 @@
+import fs from "fs-extra";
+import {
+    Branch,
+    Clone,
+    CloneOptions,
+    Commit,
+    Cred,
+    FetchOptions,
+    Object,
+    Oid,
+    Reference,
+    Remote,
+    Repository,
+    Revwalk,
+    Tag,
+} from "nodegit";
+
+import { homedir } from "os";
+import path from "path";
+import { ZERO_ADDRESS } from "..";
+
+// Problems running nodegit?
+// Try this: 'apt-get install -y python2 python3 libkrb5-dev gcc openssl libssh2-1-dev libcurl4-openssl-dev g++ make'
```

Comment:
I'm not sure how consistent the nodegit problem is, but I ran into it. We may want to make this suggestion more visible by putting it into the README.

Reply:
Great idea, added.